File: 백준/Bronze/2609. 최대공약수와 최소공배수/최대공약수와 최소공배수.py
number = list(map(int,input().split()))

for i in range(min(number),0,-1):
    if number[0]%i==0 and number[1]%i==0:
        print(i)
        break

g = max(number)
while True :
    if g % number[0] == 0 and g%number[1]==0:
        print(g)
        break
    g+=max(number)

File: 백준/Bronze/2609. 최대공약수와 최소공배수/최대공약수와 최소공배수.java
import java.io.*;
import java.util.StringTokenizer;

public class Main {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        StringTokenizer st = new StringTokenizer(br.readLine());
        int n = Integer.parseInt(st.nextToken());
        int m = Integer.parseInt(st.nextToken());
        int min = (n > m) ? m : n;
        int max = (n > m) ? n : m;

        for(int i = min ; i > 0 ; i--){
            boolean result = false;

            if(m % i == 0 && n % i == 0){
                result = true;
            }else{
                result = false;
            }

            if(result){
                System.out.println(i);
                break;
            }
        }

        int first = max;
        while(true){
            if(first % min == 0 && first % max == 0){
                System.out.println(first);
                break;
            }else{
                first += max;
            }
        }
    }
}